docs/resources/ocean_aks_np.md

```diff
@@ -276,3 +276,32 @@ The following arguments are supported:
     * `min_nics` - (Optional) Minimum number of network interfaces.
     * `min_disk` - (Optional) Minimum number of data disks available.
     * `vm_types` - (Optional, Enum `"generalPurpose", "memoryOptimized", "computeOptimized", "highPerformanceCompute", "storageOptimized", "GPU"`) The filtered vm types will belong to one of the vm types from this list.
+
+<a id="update-policy"></a>
+## Update Policy
+
+* `update_policy` - (Optional)
+  * `should_roll` - (Required) If set to true along with the cluster update, roll will be triggered.
+  * `conditioned_roll` - (Optional, Default: false) Spot will perform a cluster Roll in accordance with a relevant modification of the cluster’s settings. When set to true , only specific changes in the cluster’s configuration will trigger a cluster roll (such as availability_zones, max_pods_per_node, enable_node_public_ip, os_disk_size_gb, os_disk_type, os_sku, kubernetes_version, vnet_subnet_ids, pod_subnet_ids, labels, taints and tags).
+  * `roll_config` - (Required) While used, you can control whether the group should perform a deployment after an update to the configuration.
```

Comment:
in the code it is defined as Optional, please re-validate

Reply:
Updated to optional.